routes/cartRouter.js

```diff
@@ -1,11 +1,17 @@
 const router = require('express').Router()
 const {
-  addCart
+  addCart,
+  editCart
 } = require('../controllers/cart')
 
 // @route POST api/store/:store_id/cart
 // @desc  Add items to cart
 // @access Public
 router.post('/:store_id/cart', addCart)
 
+// @route PUT api/store/:store_id/cart
+// @desc  Add items to cart
+// @access Public
+router.put('/cart/:cart_id', editCart)
```

Comment:
confirm the route naming. is it supposed to be `/cart/:cart_id` or `/store/:cart_id`

Reply:
This is the full route `/api/store/cart/:cartId` . Let me know what you think, we can always change it if needed.

Reply:
then that's fine. i didn't check the full route